wsmux/errors.go

```diff
@@ -0,0 +1,36 @@
+package wsmux
+
+import (
+	"fmt"
+)
+
+var (
+
+	// ErrAcceptTimeout is returned when the Accept operation times out
+	ErrAcceptTimeout = fmt.Errorf("accept timed out")
```

Comment:
Errorf makes more sense for formatting... If making hardcoded errors most people use `errors.New("....")` not that's important...